src/containers/list-monitor.jsx

```diff
@@ -0,0 +1,183 @@
+import bindAll from 'lodash.bindall';
+import PropTypes from 'prop-types';
+import React from 'react';
+import VM from 'scratch-vm';
+import {connect} from 'react-redux';
+import {getEventXY} from '../lib/touch-utils';
+import {getVariableValue, setVariableValue} from '../lib/variable-utils';
+import ListMonitorComponent from '../components/monitor/list-monitor.jsx';
+
+class ListMonitor extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleActivate',
+            'handleDeactivate',
+            'handleInput',
+            'handleRemove',
+            'handleKeyPress',
+            'handleFocus',
+            'handleAdd',
+            'handleResizeMouseDown'
+        ]);
+
+        this.state = {
+            activeIndex: null,
+            activeValue: null,
+            // TODO These will need to be sent back to the VM for saving
+            width: props.width || 80,
+            height: props.height || 200
```

Comment:
I think you want `typeof props.width === 'undefined' ? 80 : props.width` so that you can set 0 for these if you need to.

Reply:
Also it is a smell to set state to the value of a prop so I think a comment is necessary to explain. Do you need a change handler from the parent component to update the width/height?  Or maybe the width/height props would be better named `initialWidth`/`initialHeight`?

Reply:
I think I explicitly do not want zero width/height, anything with zeros should get set back to the default.

I think setting the state to the props value is the standard way to deal with "transient" values (like a buffered input, etc.) The prop is "initial" i suppose, although once the w/h get sent back to the VM and propogated back here, it won't really be "initial". Once we do that (TODO comment above) we will just reset the state to the props in componentWillReceiveProps.